Added filereader Java example

lmcnulty4 opened the pull request.

Commit:
Added filereader Java example

alex952 commented on examples/java/README.md:
````diff
@@ -17,3 +17,31 @@ $ java -cp $INSTALL_PREFIX/lib/java/CdrJNI.jar:$INSTALL_PREFIX/lib/java/CodecJNI
 ```
 
 View the source [here](src/main/java/com/neueda/codec/example/App.java).
+
````
Can you remove the duplicated lines? Just leave the ones that refer to the new example.

Commit:
Renamed CdrReader to CodecReader.
Added better explanation around CodecReader.
Removed duplicated lines in README.
